core/src/env/srml/types.rs

```diff
@@ -31,14 +31,34 @@ use parity_codec::{
 
 /// The SRML fundamental types.
 #[allow(unused)]
+#[cfg_attr(feature = "test-env", derive(Debug, Clone, PartialEq, Eq))]
 pub enum DefaultSrmlTypes {}
 
+/// Private module for default Call type, so it cannot be constructed
+/// For calling into the runtime, a user defined Call type required.
+/// See https://github.com/paritytech/ink-types-node-runtime.
+mod private {
+    #[cfg_attr(feature = "std", derive(Debug, Clone, PartialEq, Eq))]
+    pub struct Call {
```

Comment:
Can't we just make it `pub enum Call { }` ? Can't be instantiated as well. Or does it cause problems?